xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3646,6 +3644,13 @@ void CVideoPlayer::SetSpeed(float speed)
     return;
   
   m_newPlaySpeed = speed * DVD_PLAYSPEED_NORMAL;
+  if (m_newPlaySpeed != m_playSpeed)
+  {
+      if (m_newPlaySpeed == DVD_PLAYSPEED_NORMAL)
+          m_callback.OnPlayBackResumed();
+      else if (m_newPlaySpeed == DVD_PLAYSPEED_PAUSE)
+          m_callback.OnPlayBackPaused();
+  }
   SetPlaySpeed(speed * DVD_PLAYSPEED_NORMAL);
```

Comment:
SetPlaySpeed() is public, so should this check occur within SetPlaySpeed()?

Side comment: i sure hope so, because razzeee hasn't updated this pr nearly enough 😉